lib/search/search_page.dart

```diff
@@ -198,105 +95,260 @@ class SearchPage extends ConsumerWidget {
-                                            child: Text(
-                                              "Profile Information",
-                                              style: Theme.of(context)
-                                                  .textTheme
-                                                  .titleLarge,
-                                            )),
-                                        Flexible(
-                                            flex: 3,
-                                            child: SingleChildScrollView(
-                                              child: Column(
-                                                children: [
-                                                  Padding(
-                                                      padding:
-                                                          EdgeInsets.all(10),
-                                                      child: ref.watch(
-                                                                  selectedRef) ==
-                                                              null
-                                                          ? Container()
-                                                          : SearchResultsItem(
-                                                              ref.watch(
-                                                                  selectedRef)!))
-                                                ],
-                                              ),
-                                            ))
-                                      ]))),
+                                  Matches(),
+                                  ProfileInformation(),
                                 ],
                               ),
                             ),
                           ),
                         ],
                       )),
 
-                  Expanded(
-                    child: Padding(
-                      padding: const EdgeInsets.symmetric(
-                          horizontal: 40.0, vertical: 20.0),
-                      child: Column(
-                        crossAxisAlignment: CrossAxisAlignment.start,
-                        children: [
-                          Container(
-                            child: Padding(
-                              padding: const EdgeInsets.all(16.0),
-                              child: Text(
-                                "Search History",
-                                style: Theme.of(context).textTheme.titleLarge,
-                              ),
-                            ),
-                          ),
-                          SearchHistory(selectedRef),
-                        ],
-                      ),
-                    ),
-                  ),
+                  SearchHistoryWidget(),
                 ])));
   }
+
+  Container SearchButton(BuildContext context, void setSearchValue()) {
```

Comment:
please follow the dart notation guidelines, and let's prefix build methods with 'build', e.g. buildSearchButton